packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -118,15 +128,16 @@ export default {
       }
     }
   },
-  
+
   methods: {
     toggleExpanded() {
       this.sublinksExpanded = !this.sublinksExpanded;
+      this.link.iHaveChildrenActive = !this.link.iHaveChildrenActive;
```

Comment:
double checking if this is expected: when nested selection is collapsed there is no hint which page is currently opened

Reply:
We should show a hint when an active item collapsed, I've added fix for that